config/initializers/custom_time_formats.rb

```diff
@@ -0,0 +1,5 @@
+# Wednesday, February 5, 2020 - 2:22 pm
+Time::DATE_FORMATS[:long_with_time] = '%A, %B %-d, %Y - %-l:%M %P'
+Date::DATE_FORMATS[:long_with_time] = '%A, %B %-d, %Y - %-l:%M %P'
+
+Time::DATE_FORMATS[:db_hm] = '%F %R'
```

Comment:
I cannot stop thinking of "db" as "database", so what is it really, here?

Reply:
It is "database". That is, this is the same as the `:db` format (Rails built-in), but with no seconds.